skada/_mapping.py

```diff
@@ -97,9 +97,11 @@ def transform(
         # thus there's no need to perform any transformations
         if X_source.shape[0] > 0:
             X_source = self.ot_transport_.transform(Xs=X_source)
-        X_adapt, _ = source_target_merge(
-            X_source, X_target, sample_domain=sample_domain
-        )
+            X_adapt, _ = source_target_merge(
+                X_source, X_target, sample_domain=sample_domain
+            )
+        else:
+            X_adapt = X_target
```

Comment:
I think the [#280](https://github.com/scikit-adaptation/skada/pull/280) solved this problem, right?